### DISCUSSION TITLE
Salt question

### DISCUSSION THREAD

**Discussion by slcw** [damionx7/Zettel-Notes-Documentation#139] [General]
The documentation makes a point to warn that uninstalling the app will make decrypting files encrypted with password encryption impossible as a unique salt is generated for each install. But, if I backup the salt and plug it in after installing the app fresh, will I be able to decrypt? It's my understanding that there are two components to the encryption: salt, and hashed password. So, as long as I know my password and provide the correct salt I should be able to decrypt. Do I have that correct, or am I missing something?

**Comment by damionx7** [comment]
Hey! This will work. And as precaution, back up your notes before attempting this on your notes.

**Comment by slcw** [comment]
Thanks!

Aug 2, 2024 12:14:58 PM Rohit ***@***.***>:
